client/src/Components/UserProfile/TextFieldInput.js

```diff
@@ -2,15 +2,16 @@ import React from 'react';
 import { makeStyles } from '@material-ui/core/styles';
 import { TextField } from '@material-ui/core'
 
-
 const useStyles = makeStyles({})
 
 export default function TextFieldInput(props) {
```

Comment:
```suggestion
export default function TextFieldInput({id, name, label, value}) {
```

if you accept your `prop` object like this into the function, the benefit of this is to
- keep track of what you need in this component
- reduce repetitive code throughout your code. therefore you can just call the variable without reaching into the object